app/forms/idv/profile_form.rb

```diff
@@ -38,5 +38,26 @@ def ssn_is_unique
         errors.add :ssn, I18n.t('idv.errors.duplicate_ssn')
       end
     end
+
+    def dob_is_sane
+      valid_date = dob_date?
+      unless valid_date && dob_in_the_past?(valid_date)
+        errors.add :dob, I18n.t('idv.errors.bad_dob')
+      end
+    end
+
+    def dob_in_the_past?(date)
+      if date > Date.today
+        false
+      else
+        true
+      end
+    end
+
+    def dob_date?
+      Date.parse(dob)
+    rescue
+      false
+    end
```

Comment:
How about this?
```ruby
def dob_is_sane
  date = parsed_dob
  unless date && dob_in_the_past?(date)
    errors.add :dob, I18n.t('idv.errors.bad_dob')
  end
end

def dob_in_the_past?(date)
  date < Date.today
end

def parsed_dob
  Date.parse(dob)
rescue
  nil
end
```
- Changes `dob_date?` from predicate to a method that either returns a valid date or `nil` as opposed to false. Predicates are meant to return either true or false, and in this case I was confused because it looked like you were passing a boolean to the `dob_in_the_past?` method
- Renames `dob_date?` to `parsed_dob` to better represent what the method does
- Renames `valid_date` to `date` because it might not always be a valid date.
- Simplifies conditional in `dob_in_the_past?`